src/scripts/youtube-info.coffee

```diff
@@ -12,12 +12,15 @@
 #
 # Notes:
 #   For text-based adapters like IRC.
+#   You will need to replace apiKey with an actual value. You can get an API key from https://console.developers.google.com/ to avoid being locked out from too many requests
```

Comment:
Use the configuration section above to list the environment variable to set with the API key